src/richie/apps/courses/templates/courses/cms/fragment_organization_glimpse.html

```diff
@@ -57,9 +57,6 @@
                 />
             {% endblockplugin %}
         </div>
-        <div class="organization-{{ organization_variant }}__content">
-            <h{{ header_level|default:3 }} class="organization-{{ organization_variant }}__title" property="name">{{ organization_page.get_title }}</h{{ header_level|default:3 }}>
-        </div>
```

Comment:
I'm not quite sure why this was removed? Maybe unrelated stuff that was commited alongside the icons/title attributes but that should be in another PR?

Anyway, I'm reverting back those changes to only have the icons/title attribute stuff in this PR.